src/server/asset_path.ts

```diff
@@ -0,0 +1,4 @@
+export let ASSET_PATH_PREFIX = "";
+export function setAssetPathPrefix(prefix: string) {
+  ASSET_PATH_PREFIX = prefix + ASSET_PATH_PREFIX;
+}
```

Comment:
It is important to have this to be able to create a `staticFile` utility where `ASSET_PATH_PREFIX` is updated on app start.